### DISCUSSION TITLE
SignalK Update

### DISCUSSION THREAD

**Discussion by MeltedPixel** [bareboat-necessities/lysmarine_gen#319] [General]
Will you be updating the package to contain SignalK 2.0.0? I tried manually updating but that did not go very well.

**Comment by mgrouch** [comment]
Yes sure

**Comment by mgrouch** [comment]
BBN OS image 2023-07-02 is now with 
SignalK 2.0.0 and NodeJS 18.

**Reply by MeltedPixel** [reply]
You rock!
